### ISSUE TITLE
Tyfon watch does not show console outputs from the server

### ISSUE THREAD

**Issue by must** [loreanvictor/tyfon#21]
When running Tyfon watch, it runs the server as well but it will not display any console.log messages or errors coming from there.

**Comment by loreanvictor** [comment]
@must I could not reproduce this. Steps that I took:

1. [Create a fresh server](https://loreanvictor.github.io/tyfon/#setup-a-server) and serve it
1. [Create a fresh client](https://loreanvictor.github.io/tyfon/#setup-a-client) and install the server SDK
1. Stop serving the server and instead run with `watch`:
    ```bash
    tyfon watch -c ../my-tyfon-client
    ```
1. Modify contents of `my-tyfon-server/index.ts`:
    ```ts
    export interface User {
      name: string;
    }
    
    export const getMessage = async (user: User) => {
      console.log('Hola!');
      return `Hellow ${user.name}!`;
    };
    ```
1. Invoke client code, so in `my-tyfon-client`, execute this:
    ```bash
    ts-node .
    ```
  As expected, I do get the log `Hola!` on the console on the server whenever I invoke the client.

**Comment by must** [comment]
I suppose this might only happen with actual errors. Can you try throwing an error on the backend?

**Comment by loreanvictor** [comment]
yes errors are not logged by default, they are instead passed to client. this is in sync with behavior in non server client contexts, i.e. when a function is called, the error is caught in the call site, and needs to be handled by the caller (for example, logged).

**Comment by must** [comment]
Yeah, that might make sense for some errors but not necessarily all and especially not in production.

**Comment by loreanvictor** [comment]
makes sense. a default logging mechanism would be pretty convenient, at least for internal server errors. I assume the standard behavior for these errors should actually change to "don't show error message to client, log it".
